server/public/styles.css

```diff
@@ -0,0 +1,73 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
```

Comment:
I don't recommend resetting all of this, `box-sizing` is fine, but only the `body` should have some excess margin you know you don't want.